resources/general.md

```diff
@@ -33,7 +33,14 @@ In your `~/.vimrc` add:
   * `set ts=2 sts=2 sw=2 expandtab`
   * `set autoindent`
 
-## No trailing whitespaces or line endings
+### Vscode
```

Comment:
I would write it as `VS Code` to use the same standard of official website

Reply:
done! c0d1649